build/charts/antrea/values.yaml

```diff
@@ -344,7 +344,7 @@ multicluster:
   enablePodToPodConnectivity: false
 
 testing:
-  ## -- enable code coverage measurement (used when testing Antrea only).
+  # -- enable code coverage measurement (used when testing Antrea only).
```

Comment:
nit: not from this PR, but all other documentation comments seem to be capitalized

Reply:
fixed